src/crypto/qat/qcccrypto.cc

```diff
@@ -539,7 +539,7 @@ bool QccCrypto::symPerformOp(int avail_inst,
     do {
       poll_retry_num = RETRY_MAX_NUM;
       if (y) {
-        yield_context yield = y.get_yield_context();
+        spawn::yield_context yield = y.get_yield_context();
```

Comment:
probably might want to move the first three change into a separate PR. they can be merged much sooner than the CMake ones. i believe.

Reply:
i'm willing to split stuff off if the cmake part gets blocked, but there are benefits to keeping them together for now - the cmake changes depend on them, and also allow us to verify that they build in CI